packages/@okta/vuepress-site/docs/guides/production-deployment/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Deployment Checklist
+---
+
+Now that you’ve built an app, get it ready to launch!
+
+By now, you might have built your first app with Okta—congrats! There are just a few final steps for you to take to customize your app and get it ready to launch. The guides below will help you take your app to the finish line.
+
+Here’s your pre-launch checklist:
+
+### Branding and Customization
+
+Ensure your application fits your brand!
+
+* [Use a custom domain](/docs/guides/custom-url-domain/overview/) - Customize your Okta organization by replacing the Okta domain name with your own domain name so all URLs look like your application.
+* [Customize and add branding to your login page](/docs/guides/custom-hosted-signin/overview/) - Tailor the login page to fit your brand or your application’s look and feel so your user experience is consistent and seamless.
+* [Customize and add branding to your error pages](/docs/guides/custom-error-pages/overview/) - If an error occurs during sign-in, Okta may need to display an error page to the user. Customize the error page to match the rest of your app.
+* [Email Domain Setup](/docs/guides/custom-email-domain/) - Update the "From" address and domain of the emails sent to your users.
+* [Customize Email Templates](https://help.okta.com/en/prod/Content/Topics/Settings/Settings_Email.htm) -  Change the look and feel of the emails sent to users by Okta.
+* [Manage access to protected resources](/docs/guides/validate-access-tokens/go/overview/) - Your app has to validate the signed tokens that are issued to end-users. Learn about this process and best practices for validating your user’s tokens.
```

Comment:
I'm confused by opening part of this sentence. Should it be "_If_ your app has to validate" or something else? It's hard to imagine it being something else, because wouldn't a sample app already have token validation in the example?